src/inspect_ai/_util/notgiven.py

```diff
@@ -15,4 +15,29 @@ def __repr__(self) -> str:
         return "NOT_GIVEN"
 
 
+def _is_notgiven(value: Any) -> bool:
+    return isinstance(value, NotGiven) or type(value).__name__ == "NotGiven"
+
+
+def sanitize_notgiven(value: Any) -> Any:
+    if value is NOT_GIVEN:
```

Comment:
Should this use `_is_notgiven`? `value is NOT_GIVEN` seems like a stricter check that is truthy only if `value` is exactly the singleton instance defined below.

If it needs to be different, it's probably worth a comment since it's not obvious.

Reply:
Oh, I understand it now